import java.util.*;
class main{
    public static void main(String[] args){
       
        try{
           int[] mynumbers={1,2,3};
            System.out.println(mynumbers[10]);
        }
        catch(Exception e){
            System.out.println("something went wrong");
        }
        finally{
               System.out.println("the try catch is finished.");
        }
        
    }
}
